app/tests/test_search.py

```diff
@@ -1406,6 +1483,15 @@ def test_search_transferring_body_with_search_filter_full_test(
 
         assert response.status_code == 200
         soup = BeautifulSoup(response.data, "html.parser")
+        select = soup.find("select", {"id": "sort"})
+        option = select.find("option", selected=True)
+        option_text = "series_id-asc"
+
+        if option:
```

Comment:
Can we remove the conditional from this test. If we need to test the scenario where option is not truthy then we can add a test for that

Reply:
I saw the updated commit but looks like we still have that condition

Reply:
@anthonyhashemi pushed a commit with the changes